core/domain/stats_jobs_one_off.py

```diff
@@ -16,58 +16,14 @@
 
 import ast
 import collections
-import datetime
-import itertools
-import json
-import re
-import sys
-import traceback
 
 from core import jobs
-from core.domain import exp_domain
-from core.domain import exp_services
-from core.domain import interaction_registry
 from core.domain import stats_jobs_continuous
-from core.domain import stats_domain
-from core.domain import stats_services
 from core.platform import models
-from extensions.interactions import base
-from extensions.objects.models import objects
 
-import utils
-
-(base_models, stats_models, exp_models) = models.Registry.import_models([
-    models.NAMES.base_model, models.NAMES.statistics, models.NAMES.exploration
+(stats_models,) = models.Registry.import_models([
+    models.NAMES.statistics
 ])
-transaction_services = models.Registry.import_transaction_services()
-
-
-def _unpack_state_rule_answer_log_model_id(item_id):
-    # Cannot unpack the item ID with a simple split, since the rule_str
-    # component can contain periods. The ID is guaranteed to always
-    # contain 4 parts: exploration ID, state name, handler name, and
-    # rule_str.
-    item_id_tokens = item_id
-
-    period_idx = item_id_tokens.index('.')
-    exp_id = item_id_tokens[:period_idx]
-
-    item_id_tokens = item_id_tokens[period_idx+1:]
-    handler_period_idx = item_id_tokens.index('submit') - 1
-    state_name = item_id_tokens[:handler_period_idx]
-
-    item_id_tokens = item_id_tokens[handler_period_idx+1:]
-    period_idx = item_id_tokens.index('.')
-    handler_name = item_id_tokens[:period_idx]
-
-    item_id_tokens = item_id_tokens[period_idx+1:]
-    rule_str = item_id_tokens
-    return (exp_id, state_name, handler_name, rule_str)
-
-
-def _get_answer_dict_size(answer_dict):
-    """Returns a size overestimate (in bytes) of the given answer dict."""
-    return sys.getsizeof(json.dumps(answer_dict))
 
 
 class StatisticsAudit(jobs.BaseMapReduceJobManager):
```

Comment:
Optional: Could you explain briefly, at a high-level, what this MR job actually does? It would be helpful for writing docstrings; I wasn't entirely sure.